ReadMe.rst

```diff
@@ -178,8 +178,8 @@ POSSIBILITY OF SUCH DAMAGE.
 
 .. CoreCI
 
-.. _Windows_VS2019: https://dev.azure.com/projectmu/mu/_build/latest?definitionId=71&&branchName=release%2F202202
-.. |WindowsCiBuild| image:: https://dev.azure.com/projectmu/mu/_apis/build/status/CI/Mu%20Common%20Intel%20MinPlatform%20CI%20VS2019?branchName=release%2F202202
```

Comment:
This change assumes the pipeline name will be changed from "Mu Common Intel MinPlatform CI VS2019" to "Mu Common Intel MinPlatform CI VS2022"